helm/templates/deployment.yaml

```diff
@@ -61,6 +61,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: AWS_ACCOUNT_ID
```

Comment:
same comment as others, this was removed in runtime v0.15.0, https://github.com/aws-controllers-k8s/code-generator/pull/220 removes this

Reply:
done